dispatcher/dispatcher.go

```diff
@@ -0,0 +1,140 @@
+package dispatcher
```

Comment:
I've named this dispatcher, but it could equally be called eventstore. Preferences?